pkg/config/config.go

```diff
@@ -157,7 +157,7 @@ type CongestionControlConfig struct {
 	NackRatioAttenuator              float64                                `yaml:"nack_ratio_attenuator,omitempty"`
 	ExpectedUsageThreshold           float64                                `yaml:"expected_usage_threshold,omitempty"`
 	UseSendSideBWE                   bool                                   `yaml:"send_side_bandwidth_estimation,omitempty"`
-	ProbeMode                        CongestionControlProbeMode             `yaml:"padding_mode,omitempty"`
+	ProbeMode                        CongestionControlProbeMode             `yaml:"probe_mode,omitempty"`
```

Comment:
correcting my stupid mistake from a long time ago. This config is not in `config-sample.yaml`. I don't think anybody is using it. So, changing name.